### ISSUE TITLE
[Feature] Can we freely decide between thousandSeparator and decimalSeparator?

### ISSUE THREAD

**Issue by faner11** [apache/echarts#18707]
### What problem does this feature solve?

I am a developer serving the Brazilian market, where thousandSeparator is '.' , decimalSeparator is ',', now it is possible to use formatter to reach the target, but it is not possible to configure it globally, which is a pain

https://github.com/apache/echarts/blob/master/src/util/format.ts#L32

### What does the proposed API look like?

const opts = {
  formatterConfig: {
    thousandSeparator: ',',
    decimalSeparator: '.',
  },
}

**Comment by helgasoft** [comment]
Try [registerLocale](https://echarts.apache.org/en/api.html#echarts.registerLocale) with [/src/i18n/langPT-br.ts](https://github.com/apache/echarts/blob/release/src/i18n/langPT-br.ts) ?

**Comment by faner11** [comment]
> Try [registerLocale](https://echarts.apache.org/en/api.html#echarts.registerLocale) with [/src/i18n/langPT-br.ts](https://github.com/apache/echarts/blob/release/src/i18n/langPT-br.ts) ?

Yes, langPT-br.ts is just a copywriting change, but the actual number display is still the standard in English, for example the number "1,000.99", in Brazil would be written as '1.000,99', I hope to achieve this effect

**Comment by helgasoft** [comment]
How about this [Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=GYIwTgBAvBAUAOBKaA-CBvAUBCYCmALgK5gB2E8AdAG4CGANkXpQQPYAyrAxg3gMoEwAS1IBzWAHJ4BALQAhAEoSANBggBbWgA8h6ouoBiYWlwJDWpACJDRQggGcAXBABMEAL6IA3Jnc_W0ubkMFg4WgCCOk4Y2DgQACa0BLTOANoSALIWKhASACpMORIA6njxRXkAFkRFRkJFfEkNRKQSALqx7sqxAJ6RQtGhcfY89HjOgkzdcRDaA-y0IHj0zkMzwKxgmgQEeGDOCMhQaGszuIQk5FRsnKP8giLiUrKKOegac3qGxqZB1rYOZwARg83liM3c4I8nWmEDYrHoZngqwgGy2SV2-1R4A8sPseyEeGiqShpxwiWSaSBAFYAAy0ygADkZqiBLgAzAyACxc1kuFxcygATiFfKBLIgQKB7OpqhFQuFoslUqBbVhMwIPXg41y9BEeBUUJw9h66hACL4QgAXjqXLT1XF6Itlij7JVWAB3CZgKaozbbTHOUCQSEQ2IdPxAA) ?
![image](https://github.com/apache/echarts/assets/13038071/bbae95aa-207a-4087-8ec1-42a06ebc0318)

**Comment by faner11** [comment]
> How about this [Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=GYIwTgBAvBAUAOBKaA-CBvAUBCYCmALgK5gB2E8AdAG4CGANkXpQQPYAyrAxg3gMoEwAS1IBzWAHJ4BALQAhAEoSANBggBbWgA8h6ouoBiYWlwJDWpACJDRQggGcAXBABMEAL6IA3Jnc_W0ubkMFg4WgCCOk4Y2DgQACa0BLTOANoSALIWKhASACpMORIA6njxRXkAFkRFRkJFfEkNRKQSALqx7sqxAJ6RQtGhcfY89HjOgkzdcRDaA-y0IHj0zkMzwKxgmgQEeGDOCMhQaGszuIQk5FRsnKP8giLiUrKKOegac3qGxqZB1rYOZwARg83liM3c4I8nWmEDYrHoZngqwgGy2SV2-1R4A8sPseyEeGiqShpxwiWSaSBAFYAAy0ygADkZqiBLgAzAyACxc1kuFxcygATiFfKBLIgQKB7OpqhFQuFoslUqBbVhMwIPXg41y9BEeBUUJw9h66hACL4QgAXjqXLT1XF6Itlij7JVWAB3CZgKaozbbTHOUCQSEQ2IdPxAA) ? ![image](https://user-images.githubusercontent.com/13038071/243132078-bbae95aa-207a-4087-8ec1-42a06ebc0318.png)

Yes, but this requires specifying the formatter in each place where it is used, which is painful to maintain when there are many charts, so I hope there is a global configuration place

**Comment by bobfw** [comment]
Still Needed

**Comment by diegoamc** [comment]
Still needed
